packages/nextjs-kit/src/components/footer.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import Link from 'next/link';
+import { Menu } from '../index';
+
+interface MenuProps {
```

Comment:
Be more specific here
```suggestion
interface FooterMenuProps {
```